certified-connectors/Mensagia/apiDefinition.swagger.json

```diff
@@ -178,48 +178,48 @@
         "x-ms-visibility": "internal"
       }
     },
-    "/agendas":{
-      "get":{
-        "responses":{
-          "200":{
+    "/agendas": {
+      "get": {
+        "responses": {
+          "200": {
             "description": "Ok",
-            "schema":{
+            "schema": {
               "type": "object",
-              "properties":{
-                "data":{
+              "properties": {
+                "data": {
                   "type": "array",
-                  "items":{
+                  "items": {
                     "type": "object",
-                    "properties":{
-                      "id":{
+                    "properties": {
+                      "id": {
                         "type": "integer",
                         "format": "int32",
                         "description": "ID",
                         "title": "ID"
                       },
-                      "name":{
+                      "name": {
                         "type": "string",
                         "description": "Name",
                         "title": "Name"
                       },
-                      "total_users":{
+                      "total_users": {
                         "type": "integer",
                         "format": "int32",
                         "description": "Total users",
                         "title": "Total Users"
                       },
-                      "busy":{
+                      "busy": {
                         "type": "integer",
                         "format": "int32",
                         "description": "Agenda is body",
                         "title": "Agenda is Busy"
```

Comment:
@Sistemes-Icon I think there's a typo in the description above. Is it correct?
Please go through all the titles and descriptions, make sure they do not look like copy paste.